src/passkey_child/passkey_child_common.c

```diff
@@ -319,3 +369,99 @@ register_key(struct passkey_data *data)
 
     return ret;
 }
+
+errno_t
+authenticate(struct passkey_data *data)
```

Comment:
high level description of what this function does could be mentioned here in a comment here, like what is in the commit message.

Reply:
I'll add that to the `passkey_child.h` since that is where I am adding all the comments.

Reply:
Added to `passkey_child.h`